package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-streams",
-  "version": "2.13.3",
+  "version": "2.13.4",
```

Comment:
Since we're releasing new functionality, this should be 2.14.0 I believe!

Reply:
Make sure the version in the package-lock.json is also updated as well!

Reply:
Yep Yep, we've got it updated both places!